### DISCUSSION TITLE
basePdf height & width

### DISCUSSION THREAD

**Discussion by rhaamo** [pdfme/pdfme#32] [Q&A] [answered]
hello,

labelmake was able to handle a basePdf height and width directly like:
```
basePdf: { width: 90, height: 38 },
```

It doesn't seems possible with pdfme, what do you recommend to just get/generate a base PDF with a specific size ?

**Comment by hand-dot** [comment, marked as answer]
@rhaamo 
Hi!

Yes, sorry about that.
I removed that option to simplify the`basePdf` interface.

You can create PDFs of any size in any software you like, such as Illustrator, and import them as `basePdf`.

**Reply by rhaamo** [reply]
I was working on using the templates to make presets to generate labels so generating basePdf by hand isn't really practical.
I will look to use PDF-Lib to generate them in that case.
